docs/manual/java/guide/devmode/ConfiguringServicesInDevelopment.md

```diff
@@ -1,3 +1,21 @@
+# How are addresses bound by services?
+
+By default, Lagom services bind to `localhost`. This address can be changed with the following.
+
+```xml
+<plugin>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-maven-plugin</artifactId>
+    <configuration>
+        <serviceAddress>0.0.0.0</serviceAddress>
+    </configuration>
+</plugin>
+```
+
+In sbt:
+
+@[service-address](code/build-services-port.sbt)
```

Comment:
This references `code/build-services-port.sbt` but the actual file name is `code/build-services-ports.sbt` (note different pluralization).

It would be a good idea to keep the Java and Scala versions as consistent as possible, except where the differences are intentional. In this case, the Scala version of the file was renamed from `build-services-ports.sbt` to `configuring-devmode-services.sbt` (reflecting the rename of the page that references it). I think the Java version should get the same rename.